src/Equinor.ProCoSys.IPO.WebApi/Authorizations/ClaimsExtensions.cs

```diff
@@ -34,7 +34,12 @@ public static string TryGetGivenName(this IEnumerable<Claim> claims)
             claim = claims.SingleOrDefault(c => c.Type == Name);
             if (claim != null)
             {
-                return claim.Value.Substring(0, claim.Value.Length - (claim.Value.Length - claim.Value.LastIndexOf(' ')));
+                var length = claim.Value.Length;
```

Comment:
why not use string.split(' ') and check for number of values found?
Also consider adding a unit test, covering different combinations of Claims (GivenName found, no GivenName found but Name with/without space)
We also have this block of code elsewhere in ProCoSys, don't remember where now, and I think we also support Name with values as this "Bjørgo, Henning"

Reply:
I can't find places where we check for values as "Bjørgo, Henning" ... do you remember this when I mention it?